cluster-controller/src/main/java/io/strimzi/controller/cluster/resources/KafkaCluster.java

```diff
@@ -130,7 +159,19 @@ public static KafkaCluster fromStatefulSet(StatefulSet ss) {
         return kafka;
     }
 
-    public ClusterDiffResult diff(StatefulSet ss, ConfigMap metricsConfigMap)  {
+    /**
+     * Return the differences between the current Kafka cluster and the deployed one
+     *
+     * @param k8s   K8SUtils client instance for accessing Kubernetes/OpenShift cluster
+     * @param namespace Kubernetes/OpenShift namespace where cluster resources belong to
+     * @param cluster   overall cluster name
+     * @return  ClusterDiffResult instance with differences
+     */
+    public ClusterDiffResult diff(K8SUtils k8s, String namespace, String cluster)  {
+
+        StatefulSet ss = k8s.getStatefulSet(namespace, cluster + KafkaCluster.NAME_SUFFIX);
+        ConfigMap metricsConfigMap = k8s.getConfigmap(namespace, cluster + KafkaCluster.METRICS_CONFIG_SUFFIX);
```

Comment:
Do we have some `getName()` variant for metrics? So that the name creations are only in one place.